rootfs/etc/systemd/system/dietpi-firstboot.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=DietPi-First_boot_installation_process
```

Comment:
Spaces are supported in the description. To match our current systemd descriptions, would prefer to see spaces used here aswell.

Reply:
fixed in 2cf3fad79ff1a3f7805c94d7d9f071661b5075a3